ada/language/ast.py

```diff
@@ -7200,7 +7211,7 @@ def is_dispatching_call():
         .. note:: This is an experimental feature. There might be some
             discrepancy with the GNAT concept of "dispatching call".
         """
-        return Entity.is_call & Let(
+        return Entity.is_direct_call & Let(
```

Comment:
1. your commit does more than what it says it does. Apparently you're also "fixing" is_dispatching/static_call.

2. I wonder if this is correct. A call to a subprogram access is a dispatching call by definition, so I wonder if it shouldn't be:

```python
Not(Entity.is_direct_call) | Let (
...
```